syft/generic/object_storage.py

```diff
@@ -161,3 +161,9 @@ def register_tags(self, obj):
 
         for tag in obj.tags:
             self._tag_to_object_ids[tag].add(obj.id)
+
+    def __len__(self):
```

Comment:
@karlhigley are you also okay with this change? to be able to call ```len``` on the object store

Reply:
Yeah, took me a minute to see what it was for, but seems pretty convenient. I think there's a `count_objects()` method on `BaseWorker` that could use this, but that's a "nice to have tweak" that shouldn't block this PR.